kubernetes/provider_test.go

```diff
@@ -439,22 +303,6 @@ func clusterVersionLessThan(vs string) bool {
 	return cv.LessThan(v)
 }
 
-type currentEnv struct {
-	Config            string
-	Ctx               string
-	CtxAuthInfo       string
-	CtxCluster        string
-	Host              string
-	User              string
-	Password          string
-	ClientCertData    string
-	ClientKeyData     string
-	ClusterCACertData string
-	Insecure          string
-	LoadConfigFile    string
-	Token             string
-}
-
```

Comment:
I removed a bunch of code from this file that appeared to be unused. Tests seem to run fine without it, but I can put it back if it looks like something we need.